protocol/src/main/proto/commands.proto

```diff
@@ -6495,12 +6519,16 @@ message Rpc {
         }
     }
 
-    message Metrics {
+    message Initial {
```

Comment:
new protocol changes should be merged into main separately. do these protos come from custom mw lib? please revert these changes.